firebase-perf/src/main/java/com/google/firebase/perf/application/AppStateMonitor.java

```diff
@@ -49,13 +49,17 @@
 public class AppStateMonitor implements ActivityLifecycleCallbacks {
 
   private static final AndroidLogger logger = AndroidLogger.getInstance();
-  private static final String FRAME_METRICS_AGGREGATOR_CLASSNAME =
-      "androidx.core.app.FrameMetricsAggregator";
 
   private static volatile AppStateMonitor instance;
-  private static boolean hasFrameMetricsAggregator = false;
+  private final boolean screenPerformanceRecordingSupported;
```

Comment:
Should this be moved to the block below (since it's no longer static, and is now final)?

Reply:
Done.